### PR TITLE
Fix #536 Update error messages for 'caliper-cli' package

### LINKED ISSUE #536: Make logging consistent
The current logging situation is a little bit ad-hoc. There is no enforced standard, and we would benefit greatly from a serviceability and usability point if we were to enforce logging standards across the Caliper code base.

We should be logging at debug level:
- [ ] entry/exit from every method, including passed values

We should be logging at error level
- [ ] every error thrown through the code

We should be logging at info level:
- [ ] caliper lifecycle items
- [ ] round information and basic statistics (we do this already)

### PR DESCRIPTION
**Pull request by Tezas-6174**
- Partially fixes #536 
- The error messages are clear and consistent.
- All the error messages are checked and updated.

### CODE REVIEW

File: packages/caliper-cli/caliper.js [hunk 1]
```diff
@@ -38,8 +38,11 @@ let results = yargs
     .strict(false)
     .argv;
 
+
+
 if (!results.thePromise) {
-    Logger.error(`Unrecognised command: '${results._[0]}'`);
+    // console.log(results._[0]);
```
Comment (davidkel) [inline packages/caliper-cli/caliper.js:44]: could you remove the commented out line

**Review by davidkel — CHANGES_REQUESTED** [review]
One minor comment from me

File: packages/caliper-cli/caliper.js (hunk 1 above)
Comment (Tezas-6174) [reply packages/caliper-cli/caliper.js]: Yes, removed it.

**Comment by davidkel** [comment]
this change can't be merged until #1352 is merged

**Review by davidkel — APPROVED** [review]
LGTM

**Pull request merged**